tooling/template-base-essentials/components/HomepageHeader.tsx

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+
+const HomepageHeader = (props: {
+  image: string | undefined;
```

Comment:
Think we can just denote the prop types as optional, e.g., `image?: string`? Not sure it makes sense to define as required then allow `undefined`; if we were to re-use this component but without one of the properties, we'd have to pass explicitly pass `undefined` (I believe).

https://kate-dev.medium.com/when-to-use-typescript-optional-property-how-is-it-different-from-declaring-property-as-undefined-2319a0ee1f07